server/events/approve_policies_command_runner.go

```diff
@@ -12,40 +12,58 @@ func NewApprovePoliciesCommandRunner(
 	commitStatusUpdater CommitStatusUpdater,
 	prjCommandBuilder ProjectApprovePoliciesCommandBuilder,
 	prjCommandRunner ProjectApprovePoliciesCommandRunner,
-	pullUpdater *PullUpdater,
+	commitOutputUpdater CommitOutputUpdater,
 	dbUpdater *DBUpdater,
 ) *ApprovePoliciesCommandRunner {
 	return &ApprovePoliciesCommandRunner{
 		commitStatusUpdater: commitStatusUpdater,
 		prjCmdBuilder:       prjCommandBuilder,
 		prjCmdRunner:        prjCommandRunner,
-		pullUpdater:         pullUpdater,
+		commitOutputUpdater: commitOutputUpdater,
 		dbUpdater:           dbUpdater,
 	}
 }
 
 type ApprovePoliciesCommandRunner struct {
 	commitStatusUpdater CommitStatusUpdater
-	pullUpdater         *PullUpdater
+	commitOutputUpdater CommitOutputUpdater
 	dbUpdater           *DBUpdater
 	prjCmdBuilder       ProjectApprovePoliciesCommandBuilder
 	prjCmdRunner        ProjectApprovePoliciesCommandRunner
 }
 
-func (a *ApprovePoliciesCommandRunner) Run(ctx *command.Context, cmd *command.Comment) {
+func (p *ApprovePoliciesCommandRunner) updateCombinedCommitStatusAndLogError(ctx *command.Context, status models.CommitStatus, cmdName fmt.Stringer) {
 	baseRepo := ctx.Pull.BaseRepo
 	pull := ctx.Pull
+	statusID, err := p.commitStatusUpdater.UpdateCombined(context.TODO(), baseRepo, pull, status, cmdName, ctx.StatusID)
+	if err != nil {
+		ctx.Log.Warnf("unable to update commit status: %s", err)
+	}
```

Comment:
Is there anything that actually uses `UpdateCombined` that actually does something with the error? If not, can we just log the error in there so that we don't have to duplicate this code everywhere?